tests/language-feature/higher-order-functions/overload-different-param-count.slang

```diff
@@ -0,0 +1,59 @@
+//TEST:SIMPLE(filecheck=CHECK): -target glsl -entry computeMain -stage compute
```

Comment:
What is this testing?

Reply:
overloads for when parameter count is different. It was one of the failures I encountered, so I added a test for it.